gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -145,15 +145,15 @@ private static void applyToJavaProject(Project project) {
                 }));
 
         project.getPluginManager().withPlugin("java-gradle-plugin", appliedPlugin -> {
-            project.getTasks().withType(JavaCompile.class).configureEach(javaCompile ->
-                    ((ExtensionAware) javaCompile.getOptions())
-                            .getExtensions()
-                            .configure(ErrorProneOptions.class, errorProneOptions -> {
-                                errorProneOptions.check("Slf4jLogsafeArgs", CheckSeverity.OFF);
-                                errorProneOptions.check("PreferSafeLoggableExceptions", CheckSeverity.OFF);
-                                errorProneOptions.check("PreferSafeLoggingPreconditions", CheckSeverity.OFF);
-                                errorProneOptions.check("PreconditionsConstantMessage", CheckSeverity.OFF);
-                            }));
+            project.getTasks().withType(JavaCompile.class).configureEach(javaCompile -> ((ExtensionAware)
+                            javaCompile.getOptions())
```

Comment:
hmm - is splitting the cast really desirable here?

Reply:
not ideal... i think i know why this is happening, the code that tries to inline a "reasonable prefix" when deciding whether to partially inline the lambda body or not, only looks at text up to the next break (which is after the cast), and since that fits, it happily accepts that inlining.

Making that logic take into account the whole first expression is an option, but might end up causing some worse formattings too down the line, i.e. if the expression inside the cast were more complicated / spans a few lines, I can see how I'd prefer this to the alternative, which would introduce not one but two indentations for the cast body.

e.g.

```java
((ExtensionAware) some
                .complicated()
                .expression())
        .getExtensions()
        ...)
```